src/styles.css

```diff
@@ -0,0 +1,200 @@
+html,body {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  overflow: hidden;
+  background-color: #e9e9e9;
+  display: flex;
+  padding: 5px 0 5px 0;
+}
+
+.nav__user {
+  display: flex;
+
+  align-items: center;
+}
+
+.nav__div {
+  margin-left: 485px;
+}
+
+.nav__div__button{
+  float: left;
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 8px 16px;
+  margin-left: 5px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.nav__div__button:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.nav__div__button:active {
+  background-color: #2196F3;
+  color: white;
+}
+
+.nav__input {
+  margin-left: 5px;
+}
+
+.nav__user {
+  margin: 0 10px 0 5px;
+}
+
+#grocery-button {
+  margin-right: 10px;
+}
+
+.body__article {
+  width: 100%;
+  height: 600px;
+  position: relative;
+}
+
+.body__article__image {
+  width: 100%;
+  height: 100%;
+}
+
+.body__article__div {
+  position: absolute;
+  height: 250px;
+  width: 250px;
+  background-color: white;
+  bottom: 100px;
+  right: 100px;
+}
+
+.body__main__section {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.body__main__section__article {
+  height: 450px;
+  width: 375px;
+  margin: 25px 0 25px 0;
+  background-color: #2196F3;
+  display: flex;
+  flex-direction: column;
+}
+
+.body__main__section__article__image {
+  max-width:100%;
+  flex: 0 1 auto;
+}
+
+.body__main__section__article__text {
+  background-color: purple;
+  height: max-content;
+  flex: 1 1 auto
+}
+
+.body__main__section__article__button {
+  flex: 0 1 40px;
+}
+
+.nav__content {
+  width: 100%;
+  height: 50px;
+  margin: 15px 0 15px 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__content__slider-wrapper {
+  font-size: 40px;
+  color:#aaa;
+  font-weight: bold;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__content__slider{
+  height: 50px;
+  padding-left:15px;
+  overflow: hidden;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+.nav__content__slider div {
+  color:#fff;
+  height: 50px;
+  margin-bottom: 50px;
+  padding: 2px 15px;
+  text-align: center;
+  box-sizing: border-box;
+}
+
+.nav__content__slider-text1 {
+  background: lightgreen;
+  animation: slide 7s linear infinite;
+}
+
+.nav__content__slider-text2 {
+  background: skyblue;
+}
+
+.nav__content__slider-text3 {
+  background: lightcoral;
+}
+
+@keyframes slide {
```

Comment:
Super cool, I feel that animations add a level of interactivity that is fun for the user.